app/controllers/rent_controller.rb

```diff
@@ -0,0 +1,15 @@
+class RentController < ApplicationController
```

Comment:
In `/app/controllers/api/v1`, comment from the previous PR to understand it

Reply:
I understand but I'll wait until the previous PR is merged to rebase this one and change this to avoid complications in the rebase.

Reply:
Ok, so it's coming soon.